src/main/java/net/wasdev/wlp/common/springboot/util/SpringBootManifest.java

```diff
@@ -57,11 +69,16 @@ String getDefault(String springBootHeaderKey) {
                     return null;
             }
         }
+
+        String getLibProvidedSuffix() {
+            return libProvidedSuffix;
+        }
     }
 
     private final String springStartClass;
     private final String springBootClasses;
     private final String springBootLib;
+    private final String springBootLibPrivided;
```

Comment:
Minor...Provided is misspelled in this variable name.

Reply:
Changed